pkg/tracingpolicy/generictracingpolicy_test.go

```diff
@@ -527,3 +529,82 @@ func TestYamlNamespaced(t *testing.T) {
 	_, ok := tp.(TracingPolicyNamespaced)
 	require.True(t, ok)
 }
+
+func createTempFile(data string) string {
+	file, err := ioutil.TempFile("", "*")
```

Comment:
As the linter said, you can use https://pkg.go.dev/os#CreateTemp here, you can add a prefix or a suffix with `tetragon` in it btw it can be useful in case the temp file fails to cleanup, not necessary since it should be in `tmp` but still it's nice. Maybe you can find other example in the codebase, not sure!